File: 백준/Bronze/9713. Sum of Odd Sequence/Sum of Odd Sequence.py
import sys
input = sys.stdin.readline

n = int(input())

for _ in range(n):
    result = 0
    for i in range(1, int(input()) + 1):
        if i % 2 != 0:
            result += i
    sys.stdout.write(str(result)+'\n')

File: 백준/Bronze/9713. Sum of Odd Sequence/Sum of Odd Sequence.java
import java.io.*;

public class Main {

    public static void main(String[] args) throws IOException {

        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));

        int n = Integer.parseInt(br.readLine());

        for (int i = 0; i < n; i++) {
            int result = 0;
            int num = Integer.parseInt(br.readLine());
            for (int j = 1; j < num+1; j++) {
                if (j % 2 != 0) result += j;
            }
            bw.write(Integer.toString(result)+'\n');
            bw.flush();
        }

    }

}
